admin_guide/manage_nodes.adoc

```diff
@@ -28,17 +28,29 @@ xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#no
 == Listing nodes
 To list all nodes that are known to the master:
 
+[source,terminal]
 ----
 $ oc get nodes
+----
+
+.Example output
```

Comment:
I believe someone noticed that we used TitleCase in 3.11 😾  - calling it out here if you want to update throughout.

Reply:
Ohhh! Good catch. Updating throughout.